web_app_drawer/views/web.xml

```diff
@@ -0,0 +1,252 @@
+              >
+        
+        <xpath expr="//a[@class='oe_logo']" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
+        
+        <xpath expr="//div[@class='oe_footer']" position="attributes">
+            <attribute name="style">display: none;</attribute>
+        </xpath>
+        
+        <xpath expr="//div[@class='oe_secondary_menus_container']/t" position="replace">
+        
+            <t t-foreach="menu_data['children']" t-as="menu">
+                <ul style="display: none" class="oe_secondary_menu nav navbar-nav" t-att-data-menu-parent="menu['id']">
+                    <t t-call="web.menu_secondary_submenu" />
+                </ul>
+            </t>
+            
+        </xpath>
+        
+    </template>
+
+    <template id="menu_secondary_submenu"
+              inherit_id="web.menu_secondary_submenu"
+              name="App Drawer - Secondary Submenu"
+              >
+        
+        <xpath expr="//ul" position="replace">
+        
+            <t t-foreach="menu['children']" t-as="menu">
+                <t t-if="menu['children']">
+                    <li t-attf-class="{{ 'dropdown-submenu' if submenu else '' }}">
+                        <t t-if="submenu">
+                            <a href="#" class="dropdown-toggle">
+                                <t t-esc="menu['name']" />
+                             </a>
+                        </t>
+                        <t t-if="not submenu">
+                            <a href="#"
+                               class="dropdown-toggle"
+                               data-toggle="dropdown"
+                               role="button"
+                               aria-haspopup="true"
+                               aria-expanded="false"
+                               >
+                                <t t-esc="menu['name']" />
+                                <span class="caret" />
+                             </a>
+                        </t>
+                        <ul t-if="menu['children']"
+                            t-attf-class="dropdown-menu oe_secondary_submenu"
+                            >
+                            <t t-call="web.menu_secondary_submenu">
+                                <t t-set="submenu" t-value="True" />
+                            </t>
+                        </ul>
+                    </li>
+                </t>
+                <t t-if="not menu['children']">
+                    <li>
+                        <t t-call="web.menu_link" />
+                    </li>
+                </t>
+            </t>
+            
+        </xpath>
+        
+    </template>
+
+    <template id="menu_link"
+              inherit_id="web.menu_link"
+              name="App Drawer - Menu Link"
+              >
+        
+        <xpath expr="//a" position="attributes">
+            <attribute name="t-att-data-menu-name">
+                menu['name']
```

Comment:
When replacing attributes, please remove all spacing. Otherwise they get harder to xpath.